apps/laboratory/src/components/AppKitHooks.tsx

```diff
@@ -64,8 +72,15 @@ export function AppKitHooks() {
         )}
 
         <Button data-testid="switch-network-hook-button" onClick={handleSwitchNetwork}>
-          Switch Network
+          {isMultichainPage ? 'Switch to EVM' : 'Switch Network'}
         </Button>
+
+        {isMultichainPage && (
+          <>
+            <Button onClick={() => switchNetwork(solana)}>Switch to Solana</Button>
+            <Button onClick={() => switchNetwork(bitcoin)}>Switch to Bitcoin</Button>
+          </>
+        )}
```

Comment:
Probably a bad pattern. Let me know if I should make a dedicated `AppKitCoreHooks` component